diff --git a/config/initializers/100-sidekiq.rb b/config/initializers/100-sidekiq.rb
index 79422e0b79..42d0f2ac29 100644
--- a/config/initializers/100-sidekiq.rb
+++ b/config/initializers/100-sidekiq.rb
@@ -56,14 +56,14 @@ else
   #
   # Instead, this patch adds a dedicated logger instance and patches
   # the #add method to forward messages to Rails.logger.
-  Sidekiq.logger = Logger.new(nil)
-  Sidekiq.logger.define_singleton_method(:add) do |severity, message = nil, progname = nil, &blk|
-    Rails.logger.add(severity, message, progname, &blk)
-  end
+  # Sidekiq.logger = Logger.new(nil)
+  # Sidekiq.logger.define_singleton_method(:add) do |severity, message = nil, progname = nil, &blk|
+  #   Rails.logger.add(severity, message, progname, &blk)
+  # end
 end
 
-Sidekiq.error_handlers.clear
-Sidekiq.error_handlers << SidekiqLogsterReporter.new
+# Sidekiq.error_handlers.clear
+# Sidekiq.error_handlers << SidekiqLogsterReporter.new
 
 Sidekiq.strict_args!
 
